src/Symfony/Component/Routing/Matcher/UrlMatcher.php

```diff
@@ -248,4 +248,11 @@ protected function getExpressionLanguage()
 
         return $this->expressionLanguage;
     }
+
+    protected function createRequest($pathinfo)
+    {
+        return class_exists('Symfony\Component\HttpFoundation\Request')
+            ? Request::create($this->context->getScheme().'://'.$this->context->getHost().$this->context->getBaseUrl().$pathinfo, $this->context->getMethod(), $this->context->getParameters())
+            : null;
```

Comment:
will do :+1: i also realized the request is probably not fully functional now; it needs some server params so it can properly resolve baseurl + basepath and such. I wanted to look at that this weekend :)